app/models/early_career_payments.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# Module namespace specific to the policy for claiming early career payments.
+#
+# TODO: Put in details of what this is for.
```

Comment:
Why can't we do this now?

Reply:
@nomasprime Could to but have missed this as was wanting to take some of the details for the translations file which was empty for the new policy at the time.

Reply:
@nomasprime have added based on best guess from this page. Overview section 1st paragraph